src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -93,10 +97,32 @@ public RestoreCommand(FrameworkName fallbackFramework)
 
             foreach (var category in ErrorMessages)
             {
-                Reports.Error.WriteLine("Errors in {0}".Red().Bold(), category.Key);
+                Reports.Error.WriteLine("{0}Errors in {1}".Red().Bold(), Environment.NewLine, category.Key);
                 foreach (var message in category.Value)
                 {
-                    Reports.Error.WriteLine("    {0}", message);
+                    Reports.Error.WriteLine($"    {message}");
+                }
+            }
+
+            var settings = Settings as Settings;
+            if (settings != null)
+            {
+                var configFiles = settings.GetNuGetConfigFiles();
+
+                Reports.Information.WriteLine($"{Environment.NewLine}NuGet Config files used:");
```

Comment:
Suggest using `Reports.Quiet` so that the extra information goes away when `--quiet` is used.